components/BookForm/BookForm.js

```diff
@@ -2,21 +2,24 @@ import Image from "next/image";
 import styled from "styled-components";
 import dummyBook from "../../public/img/dummy/dummy_book.png";
 
-export default function BookForm({ book, onEditSubmit }) {
+export default function BookForm({ book, onEditSubmit, onNewSubmit }) {
   function handleSubmit(event) {
     event.preventDefault();
     const formData = new FormData(event.target);
     const data = Object.fromEntries(formData);
-
-    onEditSubmit(data);
+    if (book) {
+      onEditSubmit(data);
+    } else {
+      onNewSubmit(data);
+    }
   }
 
   return (
     <StyledForm onSubmit={handleSubmit} id="book-form">
       <StyledTopForm>
-        {book.coverImg ? (
+        {book?.coverImg ? (
           <Image
-            src={book.coverImg}
+            src={book?.coverImg}
```

Comment:
you can shorten this part to use only one image component have conditionally decide which values are being used:

```js
      <StyledTopForm>
        <Image
          src={book ? book.coverImg : dummyBook}
          width={70}
          height={100}
          alt={book ? book.title : 'No picture'}
          priority
        />
      </StyledTopForm>
```

Reply:
Hi Niklas, but that's not how I can pass an alt="No Picture". Image wants an alt text. Or?

Reply:
Yes, Image wants an alt text, and with this ternary operator we either have the `book.title` or the text "no picture" as an alt text.

Reply:
Hi, Jörg, your function right now is doing this: 
"take the opposite value of `book` and compare it with an empty string. If that is true, return the `book.title`"
If that is working, I guess something is wrong with your `book` key. If there is no book, it should be undefinded rather than an empy string. Did you try both cases?

Reply:
You are right...
```js
      <StyledTopForm>
        <Image
          src={book.coverImg === "" ? dummyBook : book.coverImg}
          width={70}
          height={100}
          alt={book.coverImg === "" ? "No Picture" : book.title}
          priority
        />
      </StyledTopForm>
```
aber es hat funktioniert, warum auch immer. :o?

Reply:
Yeah I don't know :D

Reply:
All this did not really work. Here now the solution.

```js
      <StyledTopForm>
        <Image
          src={!book ? dummyBook : book.coverImg ? book.coverImg : dummyBook}
          width={70}
          height={100}
          alt={!book ? "No Picture" : book.coverImg ? book.title : "No Picture"}
          priority
        />
      </StyledTopForm>
```
But not really shorter or readable... ;0)